boards/raspberrypi/rpi_pico/rpi_pico_rp2040_w_defconfig

```diff
@@ -9,3 +9,23 @@ CONFIG_BUILD_OUTPUT_HEX=y
 CONFIG_UART_INTERRUPT_DRIVEN=y
 CONFIG_RESET=y
 CONFIG_CLOCK_CONTROL=y
+
+# Increase memory areas for WiFi stack
+CONFIG_MAIN_STACK_SIZE=4096
+CONFIG_HEAP_MEM_POOL_SIZE=16384
+CONFIG_SHELL_STACK_SIZE=2560
+
+# Default networking configuration
+CONFIG_WIFI=y
+CONFIG_NETWORKING=y
+CONFIG_NET_L2_ETHERNET=y
+CONFIG_NET_IPV6=n
+CONFIG_NET_IPV4=y
+CONFIG_NET_DHCPV4=y
```

Comment:
These options belong to application and not to board default, I suggest these are removed.

Reply:
Thanks for pointing that out.  You are quite correct:  none of the CONFIG_NET* options belong in the defconfig.